### PR TITLE
feat(browser): Add getInfo to make browser detection async

### PR DESCRIPTION
**Pull request by FranklinWaller**

### CODE REVIEW

File: src/js/utils/Browser.js [hunk 1]
```diff
@@ -1,6 +1,51 @@
...
 
-const Browser = {};
+const Browser = {
+    /**
+     * Retrieves browser info in a asynchronous manner
+     * Please use this instead of accessing properties directly.
+     *
+     * @returns {Promise<object>}
+     */
+    async getInfo() {
```
Comment (Thomasvdam) [inline src/js/utils/Browser.js:11]: I'm thinking we should be caching this.

File: src/js/utils/services/getTinyMp4.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
...AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADengAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAcAAAAsJtb292AAAAbG12aGQAAAAAAAAAAAAAAAAAAAPoAAAALwABAAABAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAAAB7HRyYWsAAABcdGtoZAAAAAMAAAAAAAAAAAAAAAIAAAAAAAAALwAAAAAAAAAAAAAAAQEAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAACRlZHRzAAAAHGVsc3QAAAAAAAAAAQAAAC8AAAAAAAEAAAAAAWRtZGlhAAAAIG1kaGQAAAAAAAAAAAAAAAAAAKxEAAAIAFXEAAAAAAAtaGRscgAAAAAAAAAAc291bgAAAAAAAAAAAAAAAFNvdW5kSGFuZGxlcgAAAAEPbWluZgAAABBzbWhkAAAAAAAAAAAAAAAkZGluZgAAABxkcmVmAAAAAAAAAAEAAAAMdXJsIAAAAAEAAADTc3RibAAAAGdzdHNkAAAAAAAAAAEAAABXbXA0YQAAAAAAAAABAAAAAAAAAAAAAgAQAAAAAKxEAAAAAAAzZXNkcwAAAAADgICAIgACAASAgIAUQBUAAAAAAfQAAAHz+QWAgIACEhAGgICAAQIAAAAYc3R0cwAAAAAAAAABAAAAAgAABAAAAAAcc3RzYwAAAAAAAAABAAAAAQAAAAIAAAABAAAAHHN0c3oAAAAAAAAAAAAAAAIAAAFzAAABdAAAABRzdGNvAAAAAAAAAAEAAAAsAAAAYnVkdGEAAABabWV0YQAAAAAAAAAhaGRscgAAAAAAAAAAbWRpcmFwcGwAAAAAAAAAAAAAAAAtaWxzdAAAACWpdG9vAAAAHWRhdGEAAAABAAAAAExhdmY1Ni40MC4xMDE=';
```
Comment (Thomasvdam) [inline src/js/utils/services/getTinyMp4.js:12]: We're insane. :D

**Review by Thomasvdam — COMMENTED** [review]
I think we should cache the result of the getInfo call as it's unlikely to change, but it's not a deal breaker seeing how quick it is. (Though we should double check on older mobile devices)

File: src/js/utils/Browser.js (hunk 1 above)
Comment (FranklinWaller) [reply src/js/utils/Browser.js:11]: Yes, since it's 22 nano seconds i think we for now can delay this.

**Pull request merged**